src/common/providers/https.ts

```diff
@@ -500,6 +504,55 @@ interface CallableTokenStatus {
   auth: TokenStatus;
 }
 
+function decodeToken(token: string): unknown {
```

Comment:
I really like the "unsafe" prefix in other places. Maybe we should mark this the same as well so future authors don't blindly export & use it.